### PR TITLE
Cmake fix

### PR DESCRIPTION
**Pull request by biddisco**
A couple of minor cmake fixes to get it building locally

### CODE REVIEW

**Comment by jgphpc** [comment]
```
[  PASSED  ] Ran 33/34 test case(s) from 34 check(s) (0 failure(s), 1 skipped)
```

**Review by jgphpc — APPROVED** [review]

File: src/evrard/CMakeLists.txt [hunk 1]
```diff
@@ -1,10 +1,16 @@
+# this target uses cuda but isn't compiled with cuda compiler
+# so add the right cuda runtime lib
+find_library(CUDART_LIBRARY cudart ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})
```
Comment (sekelle) [inline src/evrard/CMakeLists.txt:3]: Let's add `find_package(CUDAToolkit)` just after we enable the CUDA language. Then we can
link applications against `CUDA::cudart` and omit adding the include dirs explicitly.

**Review by sekelle — CHANGES_REQUESTED** [review]

File: src/evrard/CMakeLists.txt (hunk 1 above)
Comment (biddisco) [reply src/evrard/CMakeLists.txt:3]: I actually tried that, but for reasons I didn't want to spend (waste) time trying to understand, `find_package(CUDAToolkit)` didn't work on my laptop, so I left it as find library for now. I might have made a silly typo or something and should try again really.

**Comment by sekelle** [comment]
superseded by #109

**Pull request closed without merging**